genesyscloud/routing_utilization/resource_genesyscloud_routing_utilization_test.go

```diff
@@ -108,30 +109,33 @@ func TestAccResourceRoutingUtilizationWithLabels(t *testing.T) {
 		greenLabelName     = "Terraform Green Label" + uuid.NewString()
 	)
 
-	CleanupRoutingUtilizationLabel()
+	err := CleanupRoutingUtilizationLabel()
+	if err != nil {
+		return
+	}
```

Comment:
Might as well log the error here with t.Skip(""). Makes it easier to understand what happened in the test